sormas-backend/src/main/java/de/symeda/sormas/backend/person/PersonFacadeEjb.java

```diff
@@ -284,11 +288,11 @@ public JournalPersonDto getPersonForJournal(String uuid) {
 		if (detailedPerson != null) {
 			JournalPersonDto exportPerson = new JournalPersonDto();
 			exportPerson.setUuid(detailedPerson.getUuid());
-			exportPerson.setEmailAddress(detailedPerson.getEmailAddress());
+			exportPerson.setEmailAddress(detailedPerson.getEmailAddress(false));
 			if (configFacade.getPatientDiaryConfig().isActive()) {
 				try {
 					PhoneNumberUtil phoneUtil = PhoneNumberUtil.getInstance();
-					Phonenumber.PhoneNumber numberProto = phoneUtil.parse(detailedPerson.getPhone(), "DE");
+					Phonenumber.PhoneNumber numberProto = phoneUtil.parse(detailedPerson.getPhone(false), "DE");
```

Comment:
This one can now throw a SeveralNonPrimaryContactDetailsException as well, or? SORMAS should catch it here.

Reply:
Also, when catching the NumberParseException a few linew below, and in the else { } below that, the SeveralNonPrimaryContactDetailsException can be thrown.

Reply:
I case of a SeveralNonPrimaryContactDetailsException being thrown, null should be returned.

Reply:
It should not be the case, as all persons registered in journals should have the correct contact information.

Reply:
It can be the case, but the new commits take this into account.